cirq-superstaq/requirements.txt

```diff
@@ -1,3 +1,4 @@
 cirq>=1.0.0
 general-superstaq~=0.4.26
+qiskit>=0.43.3
```

Comment:
we don't want css to require qiskit by default. it's already required in `example-requirements` so we should have it available for tests anyway - see similar tests in compiler_output_test.py